mkosi/resources/man/mkosi.1.md

```diff
@@ -888,6 +888,10 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     in the ESP for compatibility with signed versions of grub which load
     the grub configuration from this location.
 
+    Note: When using `systemd-boot` or `systemd-boot-signed`, the image
+    must include `bootctl`. This usually requires adding a package, e.g.
+    `systemd-boot` on Debian-based distributions or `systemd` on Arch.
```

Comment:
Isn't it hard not to have `systemd` on Arch? How about
```suggestion
    must include `bootctl`. This may require adding a package when it is packaged separately, e.g.
    `systemd-boot` on Debian-based distributions.
```